api/productions.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable no-console */
```

Comment:
Instead of disabling this rule for the whole file, it might be better to create some kind of wrapper function around `console.log()` that takes the same arguments and passes them to console.log, but first checks if the app is running in development mode and if it isn't it doesn't do anything.

Something like:
```
debug => (...args) {
  if (isDevelopmentMode) { // See my other comment about `process.env` variables
    /* eslint-disable no-console */
    console.log(...args);
    /* eslint-enable no-console */
  }
}
```